src/main/java/com/tvtracker/enterprise/dao/BaseDAO.java

```diff
@@ -257,7 +258,7 @@ public boolean delete() throws SQLException, IOException, ClassNotFoundException
      * @return ArrayList of key value pairs
      */
     private ArrayList<HashMap<String, Object>> getResultList(ResultSet resultSet) throws SQLException {
-        ArrayList<HashMap<String, Object>> results = new ArrayList<HashMap<String, Object>>();
+        ArrayList<HashMap<String, Object>> results = new ArrayList<>();
```

Comment:
does not need to be explicitly defined twice. Works with just <>.
This improves readability and reduces clutter.
Source: https://jsparrow.github.io/rules/diamond-operator.html#description